python/ray/tune/commands.py

```diff
@@ -154,6 +158,11 @@ def list_trials(experiment_path,
         checkpoints_df["logdir"] = checkpoints_df["logdir"].str.replace(
             experiment_path, '')
 
+    if filter:
```

Comment:
`filter` is a python built-in function so I would use some other name here, for example `filter_string`.